src/web/mjs/connectors/mundowuxia.mjs

```diff
@@ -1,12 +0,0 @@
-import WordPressMadara from './templates/WordPressMadara.mjs';
-
-export default class MundoWuxia extends WordPressMadara {
```

Comment:
Consider to remove website in a separate pull request (also remove the corresponding icon)